projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.ts

```diff
@@ -17,15 +19,29 @@ export class ReadQueryManagementFieldComponent extends AbstractFieldReadComponen
   public query: QueryListItem;
   public showQueryList: boolean = true;
   public caseId: string;
+  public messageType : string;
+
+  public followUpQuery: string = QueryCreateContext.FOLLOWUP;
+  public respondToQuery: string = QueryCreateContext.RESPOND;
+
+  public isQueryClosed: boolean = false;
+
+  public value: boolean;
+  public isMultipleFollowUpEnabled: boolean;
+
+  public enableServiceSpecificMultiFollowups: string[];
 
   constructor(private readonly route: ActivatedRoute,
     private sessionStorageService: SessionStorageService,
-    private readonly caseNotifier: CaseNotifier
+    private readonly caseNotifier: CaseNotifier,
+    private readonly abstractConfig: AbstractAppConfig,
   ) {
     super();
   }
 
   public ngOnInit(): void {
+    this.enableServiceSpecificMultiFollowups = this.abstractConfig.getEnableServiceSpecificMultiFollowups() || [];
+    this.isMultipleFollowUpEnabled = this.enableServiceSpecificMultiFollowups.some((jurisdiction) => jurisdiction === this.caseNotifier?.cachedCaseView?.case_type?.jurisdiction.id);
```

Comment:
Should be listening to the caseNotifier here, rather than checking it's internal state